packages/web3-providers/src/opensea/index.ts

```diff
@@ -61,13 +61,14 @@ function createERC721TokenFromAsset(
     chainId: ChainId,
     asset: OpenSeaResponse,
 ): ERC721TokenDetailed {
+    const imageURL = asset?.image_url ?? asset?.image_preview_url ?? asset?.image_original_url
```

Comment:
```suggestion
    const imageURL = asset?. image_original_url ?? asset?. image_url ?? asset?.image_original_url ?? ''
```

Is priority `image_url > image_preview_url > image_original_url` ?